src/stagehandler.ts

```diff
@@ -281,8 +288,15 @@ export class StageHandler {
 			reply.errcode = response.errcode;
 			reply.error = response.error;
 			res.status(STATUS_UNAUTHORIZED);
-			// eslint-disable-next-line no-magic-numbers
-			this.log.debug(`Reply=\n${JSON.stringify(reply, null, 2)}, STATUS CODE: ${STATUS_UNAUTHORIZED}`);
+			// ****************************************************************************************************
+			// CAUTION! It is not safe to log such a things since it may disclose the configuration secrets.
+			// Use it on your own risk (only for the debugging) and don't forget to comment out for the production.
+			// If doing so, remove also the "XXX" in the next line to make the linter happy.
+			//
+			// XXXeslint-disable-next-line no-magic-numbers
+			// this.log.silly(`Reply=\n${JSON.stringify(reply, null, 2)}, STATUS CODE: ${STATUS_UNAUTHORIZED}`);
+			// ****************************************************************************************************
+
```

Comment:
Same here

Reply:
Well, that lines was crucial to figure out the cause of the initial bug. That's why I decided to keep they commented out, just for the case if somebody else will have to do the same. But it would be also not a big deal to just throw it away.

Reply:
If the lines are kept, I think the comment can be a lot cleaner
```suggestion
			// The following lines should only be uncommented for debugging, since the reply may include sensitive data.
			//
			// // eslint-disable-next-line no-magic-numbers
			// this.log.silly(`Reply=\n${JSON.stringify(reply, null, 2)}, STATUS CODE: ${STATUS_UNAUTHORIZED}`);
```

Reply:
Too late, I just threw them away 😃